### DISCUSSION TITLE
On boot exit 2 after that no problem

### DISCUSSION THREAD

**Discussion by wassereimer86** [nginx-proxy/nginx-proxy#1796] [Q&A] [answered]
Hi,

i hope that somebody can help me. When i reboot/start my server, all Containers start, except nginx-proxy.  There i get an "Exited (2)" in the status. After i make an "docker-compose up -d" it starts und runs without any problems. Does anybody has ideas what the reason can be? The logs don´t tell my anything (at least for me as a docker beginner).

`SIGQUIT: quit
PC=0x46b9e1 m=0 sigcode=0

goroutine 34 [syscall]:`

And than a lot of goroutine´s.

Thank you for any answers. 🙂

![WindowsTerminal_2021-10-09_19-43-58](https://user-images.githubusercontent.com/22779744/136670145-490e28b5-6d94-4dd0-856e-505da4640cc3.png)

![WindowsTerminal_2021-10-09_19-46-12](https://user-images.githubusercontent.com/22779744/136670146-836f576a-940b-4d87-ade8-e3cd07701584.png)

[log.txt](https://github.com/nginx-proxy/nginx-proxy/files/7316176/log.txt)

**Comment by Murazaki** [comment]
Same issue on my side with evertramos/nginx-proxy-automation stack.
 I wonder if the nginx-proxy container could have stored a SIGQUIT signal and run it on startup ?

**Reply by Murazaki** [reply]
(Running a fresh Debian 11 bullseye, docker 20.10.9, docker-compose 1.29.2)

**Comment by wassereimer86** [comment, marked as answer]
See https://github.com/nginx-proxy/nginx-proxy/issues/1809

**Reply by buchdag** [reply]
@wassereimer86 sorry I saw and marked this discussion but completely forgot to answer 😐

**Reply by wassereimer86** [reply]
Absolutly no problem. I´m thankful for your work. You have to do more than enough. And this is the discussion area not the bug/problem area. :) I wasn´t sure if it´s my problem and if i missed something. So i tried it here before saying it´s a bug or something like that. :)
